LondonBritishLibrary/orient/BLorient8823.xml

```diff
@@ -0,0 +1,266 @@
+                                <date notBefore="1700" notAfter="1799" resp="PRS8999Strelcyn"/>
+                            </handNote>
+                            <handNote script="Ethiopic" xml:id="h2">
+                                <locus from="45r" to="103r"/>
+                                <desc>Good hand, large characters.</desc>
+                                <date notBefore="1700" notAfter="1799" resp="PRS8999Strelcyn"/>
+                            </handNote>
+                            <handNote xml:id="h3" script="Ethiopic">
+                                <locus from="71ra" to="103ra"/>
+                                <desc>Good hand, large characters, especially from <locus from="95r" to="103r"/>.</desc>
+                            </handNote>
+                        </handDesc>
+                        <decoDesc>
+                            <decoNote xml:id="d1" type="other">
+                                <locus target="#14 #29 #40 #63 #67 #98"/>
+                                <desc>The different parts of the manuscript are marked by small pieces of red yarn sewn onto the margins of the 
+                                    relevant leaves.</desc>
+                            </decoNote>
+                        </decoDesc>
+                        <additions>
+                            <list>
+                                <item xml:id="e1">
+                                    <desc type="OwnershipNote">Note inside front cover.</desc>
+                                    <q xml:lang="en"><persName role="owner" ref="PRS3244Curzon">R. CVRZON</persName></q>
+                                </item>
+                                <item xml:id="e2">
+                                    <locus target="103v"/>
+                                    <desc type="DonationNote"/>
+                                    <q xml:lang="en">Bequeathed by 
+                                        <persName role="bequeather" ref="PRS14320DareaCurzon">Darea, Baroness Zouche</persName>.
+                                        <date when="1917-10-13">13 Oct. 1917</date>.</q>
+                                </item>
+                                <item xml:id="e3">
+                                    <locus target="#2r #32bis"/>
+                                    <desc>Pen trials</desc>  
+                                </item>
+                                <item xml:id="e4">
+                                    <locus target="#2r"/>
+                                    <desc type="Inventory">Note on the contents of the manuscript.</desc>
+                                </item>
+                                <item xml:id="e5">
+                                    <locus target="#32bis"/>
+                                    <desc type="Supplication">Beta Israel eulogistic formula</desc>
+                                    <q xml:lang="gez">ይትባረክ፡ እግዚአብሔር፡ አምላከ፡ ፳ኤል፡</q>
```

Comment:
just wondering whether it is worth to mark up abbreviations (እስራኤል)

Reply:
If I use አምላከ፡ ፳ኤል<abbr>እስራኤል፡</abbr> it will turn out: አምላከ፡ ፳ኤል(እስራኤል)፡